.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 lib
 es
+.idea
```

Comment:
As I said haha. So we will remove it later. Btw, `.* ` in .npmignore avoid publishing them on the registry.